src/adapters/supabase/helpers/client.ts

```diff
@@ -136,18 +148,30 @@ export const upsertIssue = async (issue: Issue, additions: IssueAdditions): Prom
 export const upsertUser = async (user: UserProfile): Promise<void> => {
   const logger = getLogger();
   const { supabase } = getAdapters();
-  const { data, error } = await supabase.from("users").select("id").eq("user_login", user.login).single();
+  const { data, error } = await supabase.from("users").select("id").eq("user_login", user.login);
```

Comment:
this line is causing a problem when analytics are enabled.

From local console
```
ERROR (event): Checking user failed, error: {"code":"42703","details":null,"hint":null,"message":"column users.id does not exist"}
    Error: Checking user failed, error: {"code":"42703","details":null,"hint":null,"message":"column users.id does not exist"}
        at upsertUser
```
from mezmo
```
 UbiquiBot ERROR column users.id does not exist
```

Reply:
this error should be fixed now but I can't know for certain because I'm discovering some other bugs with analytics